app/scripts/modules/netflix/pipeline/stage/properties/create/propertyExecutionDetails.controller.js

```diff
@@ -17,7 +17,7 @@ module.exports = angular.module('spinnaker.netflix.pipeline.stage.property.detai
     let initialized = () => {
       $scope.detailsSection = $stateParams.details;
       $scope.properties = getPropertiesForAction($scope.stage.context);
-      $scope.originalProperties = extractProperties($scope.stage.context.originalProperties);
+      $scope.originalProperties = extractProperties($scope.stage.context.originalProperties || []);
```

Comment:
or one-line it (seems unlikely we'll have something that's both truthy and not an array in `$scope.stage.context.originalProperties`), e.g.
```javascript
const extractProperties = (propertyList) => propertyList ? propertyList.map(prop => prop.property) : [];
```